stdlib/REPL/src/latex_symbols.jl

```diff
@@ -49,7 +49,7 @@ end
 # symbol listing, using the following script:
 #=
 fname = "unicode-math-table.tex"
-isfile(fname) || download("http://mirror.math.ku.edu/tex-archive/macros/latex/contrib/unicode-math/$fname", fname)
+isfile(fname) || download("https://mirror.math.ku.edu/tex-archive/macros/latex/contrib/unicode-math/$fname", fname)
```

Comment:
```suggestion
isfile(fname) || download("https://ctan.org/tex-archive/macros/latex/contrib/unicode-math/$fname", fname)
```
+ (auto redirect to the nearest mirror)